.travis.yml

```diff
@@ -77,7 +77,23 @@ jobs:
     node_js: 9.4.0
     script:
     - travis_retry scripts/setup-minikube-for-linux.sh
-    - make -C enterprise-suite-latest frontend-tests
+    - make -C enterprise-suite-latest frontend-tests1
+    after_script:
+      # Cypress videos.
+      - artifacts upload $(find . -name '*.mp4')
+      - artifacts upload $(find . -name '*.png')
+
+  - name: Frontend E2E Tests 2 - Minikube Latest
+    # Only build latest in the daily cron - don't block PRs or master releases.
+    if: type in (cron)
```

Comment:
I wonder if the common stuff can be included only once somehow.  My Travis-foo isn't good enough to know...